SCClassLibrary/Common/GUI/tools/Stethoscope.sc

```diff
@@ -437,6 +460,10 @@ Stethoscope {
 		);
 		^true;
 	}
+	
+	bounds_ { arg rect;
```

Comment:
Please [use pipes](https://github.com/supercollider/supercollider/wiki/Style-Guidelines%3A-SuperCollider#rule-use-pipes-instead-of-the-arg-keyword-to-express-parameter-lists).